fluentnamer/src/main/java/com/azure/autorest/fluent/transformer/FluentTransformer.java

```diff
@@ -43,6 +43,7 @@ public CodeModel preTransform(CodeModel codeModel) {
     public CodeModel postTransform(CodeModel codeModel) {
         codeModel = new OperationGroupFilter(fluentJavaSettings.getJavaNamesForRemoveOperationGroup()).process(codeModel);
         codeModel = new NamingConflictResolver().process(codeModel);
+        codeModel = new OperationGroupRenamer(fluentJavaSettings.getJavaNamesForRenameOperationGroup()).process(codeModel);
```

Comment:
should be before `NamingConflictResolver`?

Reply:
Yeah, makes sense. If we have two conflict operation groups and want to rename, we probably want to rename both.